tests/certbot-boulder-integration.sh

```diff
@@ -281,7 +281,7 @@ CheckCertCount() {
 }
 
 CheckGroup() {
-    group_mode() { echo $((0`stat -c %a $1` & 070)); }
+    group_mode() { echo $((0`stat -c %a $1` & 074)); }
```

Comment:
Name of the method and error message should change, as it check know both permission on group and read permission on world.